scalalogging/src/main/scala/com/typesafe/scalalogging/Logger.scala

```diff
@@ -17,8 +17,8 @@ object Logger {
   /**
    * Create a [[LoggerTakingImplicit]] wrapping the given underlying `org.slf4j.Logger`.
    */
-  def takingImplicit[A](underlying: Underlying)(implicit ev: CanLog[A]): LoggerTakingImplicit[A] =
-    new LoggerTakingImplicit[A](underlying)
+  // def takingImplicit[A](underlying: Underlying)(implicit ev: CanLog[A]): LoggerTakingImplicit[A] =
+  //   new LoggerTakingImplicit[A](underlying)
```

Comment:
this could be related to https://github.com/lampepfl/dotty/issues/16630

Reply:
Yes, I think that is what caused me to have to remove these methods.

Reply:
I have raised a PR on scalac to fix this bug, however it's not clear if this is the correct fix as I never looked into the compiler before and it seems to be very complicated.  https://github.com/lampepfl/dotty/pull/18663